src/components/Person.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../App.scss';
+
+const Person = ({ name, age, sex, isMarried, partnerName }) => {
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="person">
+      <h2 className="person__name">{`My name is ${name}`}</h2>
```

Comment:
Let's do it with proper formatting fix it in all <p> tags here
```suggestion
      <h2 className="person__name">
        {`My name is ${name}`}
      </h2>
```